packages/server/src/api/controllers/Contacts/Contacts.ts

```diff
@@ -153,7 +153,7 @@ export default class ContactsController extends BaseController {
       check('email')
         .optional({ nullable: true })
         .isString()
-        .normalizeEmail()
+        .normalizeEmail({ gmail_remove_dots: false })
```

Comment:
Let us remove the `normalizeEmail` function entirely, we want to give the user freedium to enter any email as long was email address.

Reply:
pushed new changes, i removed normalizeEmail function as requested.